addons/mail/static/src/owl/components/moderation_reject_dialog/moderation_reject_dialog.xml

```diff
@@ -4,8 +4,8 @@
         <div><!-- TODO FIXME necessary div for useRef, can be removed at next owl update, see https://github.com/odoo/owl/issues/638 -->
         <t t-set="title">Send explanation to author</t>
         <Dialog contentClass="'o_ModerationRejectDialog'" title="title" size="'medium'" t-ref="dialog">
-            <input class="form-control" type="text" placeholder="Subject" autofocus="autofocus" t-model="state.title"/>
-            <textarea class="form-control mt16" placeholder="Mail Body" t-model="state.comment"/>
+            <input id="o_ModerationRejectDialog_title" class="form-control" type="text" placeholder="Subject" autofocus="autofocus" t-model="state.title"/>
+            <textarea id="o_ModerationRejectDialog_comment" class="form-control mt16" placeholder="Mail Body" t-model="state.comment"/>
```

Comment:
why `id` instead of `class`?

Reply:
Habit for form elements to be able to use "label for".

Reply:
I would still not use `id` and prefer `class`

Reply:
If it's the only change you can do it and merge I suppose.